_posts/2017-08-21-police-malpractice.md

```diff
@@ -0,0 +1,50 @@
+
+[^2]: In Minnesota alone, police paid out $1.8M after killing [Dominic Felder](https://www.mprnews.org/story/2010/10/25/excessive-force-verdict) in 2006, $200k after killing [Jamal Clark](https://www.mprnews.org/story/2019/08/09/attorneys-for-jamar-clark-family-members-confirm-agreement-on-200000-settlement) in 2014, and $3M after killing [Philando Castile](https://en.wikipedia.org/wiki/Shooting_of_Philando_Castile) in 2017.
+
+One path forward is to require each officer to hold liability insurance, rather than being covered by the department or city. The officers who killed George Floyd were [repeatedly reprimanded](https://www.mprnews.org/story/2015/11/19/the-officers-in-the-jamar-clark-shooting) for excessive force. The officers who killed Jamal Clark had a [history of violence](https://www.mprnews.org/story/2015/11/19/the-officers-in-the-jamar-clark-shooting) as well. Each of these men was a multimillion-dollar settlement waiting to happen, which any insurance underwriter could have seen. Prohibitive insurance costs should have made them unemployable as police officers long before they had a chance to kill, regardless of support from the department or union.
+
+<!-- the cost of insurance could also be shared by superior officers, creating an incentive not to hire high-risk officers. but that's a next-order effect, and explaining might be more of a distraction than it's worth -->
+
+Addressing police misconduct through civil suits and liability insurance may also help shift public opinion. Under [today's laws](https://en.wikipedia.org/wiki/Qualified_immunity), it's [hard to convict the police](https://www.politico.com/news/2020/06/01/minnesota-attorney-general-hard-to-convict-police-293920). They're [rarely even indicted](https://fivethirtyeight.com/datalab/ferguson-michael-brown-indictment-darren-wilson/). But criminal cases attract far more attention than civil suits (see chart below). A [consistent majority](http://www.gallup.com/poll/213869/confidence-police-back-historical-average.aspx) of Americans report a great deal of confidence in the police. To advance reforms at the ballot box, that confidence needs to be shaken. Putting million-dollar settlements front and center might do the trick. "[Officer who shot Philando Castile found not guilty on all counts](http://www.cnn.com/2017/06/16/us/philando-castile-trial-verdict/index.html)" won't.
```

Comment:
Let's try to draw out the logic, because I don't think it flows well right now.

> People ignore civil suits -> People Like police -> ??? -> People don't like police

I think you are trying to argue: People currently like police, partially because they ignore civil law suits. But if we made criminal lawsuits not possible, and civil more common/large, people would pay attention to those.

I think you have some unstated parts of your argument (like if there are no criminal suits), and I don't know if I agree with that conclusion.